src/firmware/app/primitives/BUILD

```diff
@@ -102,14 +102,22 @@ cc_library(
     srcs = ["primitive.c"],
     hdrs = ["primitive.h"],
     deps = [
-        "//firmware/main/app/world:firmware_world",
+        "//firmware/app/world:firmware_world",
     ],
 )
 
 cc_library(
     name = "primitive_manager",
     srcs = ["primitive_manager.c"],
     hdrs = ["primitive_manager.h"],
+    copts = select({
+        # TODO: this should not be dependent on freertos at all
+        "//cc_toolchain:cpu_stm32f4": [
+            "-isystem firmware/boards/shared/legacy_freertos/include",
+            "-isystem firmware/boards/shared/legacy_stm32lib/include",
+        ],
+        "//conditions:default": [],
```

Comment:
what does this do?

Reply:
`"//conditions:default": [],` just means do nothing by default.

The whole thing is because we had to add includes like this due to the old structure for FreeRTOS stuff. It will be going away super soon, this is somewhat of a halfway PR just to reduce the review load.